Generate releases (#16)
kyma-project/bundles: bundles/redis-0.0.3/README.md

## Overview

> The redis bundle is an example of bundle. It wraps official stable [redis chart](https://github.com/kubernetes/charts/tree/master/stable/redis).

Reviewer: ```suggestion
The Redis bundle is an example of a bundle. It wraps the official stable [Redis chart](https://github.com/kubernetes/charts/tree/master/stable/redis).
```

> You can find more information about redis chart in [the chart documantation](chart/redis/README.md)

Reviewer: ```suggestion
Find more information about the Redis chart in the chart's [documentation](chart/redis/README.md).
```
